internal/agent/agent.go

```diff
@@ -48,25 +54,55 @@ func stackTriger() {
 	signal.Notify(sigchain, syscall.SIGUSR1)
 }
 
-func NewAgent(preferDriver string, allocAllDevice bool, devicepluginPreStart bool) (*Agent, error) {
+func NewAgent(preferDriver string, allocAllDevice bool, devicepluginPreStart bool, simpleMode bool, exposeERIs string) (*Agent, error) {
 	kubernetes, err := k8s.NewKubernetes()
 	if err != nil {
 		return nil, err
 	}
+	agentLog.Info("NewAgent: ", "simpleMode", simpleMode)
 	return &Agent{
 		kubernetes:           kubernetes,
 		driver:               drivers.GetDriver(preferDriver),
 		allocAllDevices:      allocAllDevice,
 		devicepluginPreStart: devicepluginPreStart,
+		simpleMode:           simpleMode,
+		exposeERIs:           strings.Split(exposeERIs, ","),
```

Comment:
The `exposeERIs` parameter is preferably configurable via regular expressions, because the number and names of erdma interfaces can vary across different nodes within cluster.